README.md

```diff
@@ -3,8 +3,8 @@
 Contributors: lukecarbis, ryankienstra, Stino11, rheinardkorf
 Tags: Gutenberg, Blocks
 Requires at least: 5.0
-Tested up to: 5.2.1
-Requires PHP: 5.4
+Tested up to: 5.2
+Requires PHP: 5.6
```

Comment:
It'll be important to note the bumped PHP version in the Changelog for the next release.